### DISCUSSION TITLE
RAK3172 with LoRaMac-Node

### DISCUSSION THREAD

**Discussion by gilbertoe** [Lora-net/LoRaMac-node#1253] [Q&A]
Hi

Does someone have already tried to port the LoRaMac-Node stack to the RAK3172 module. It uses an STM32WLE5 chip.

I would to use the  RAK with a custom firmware.

BR
Gilberto

**Comment by x893** [comment]
Hi
Lora-Net + ST example + info from RAK = SUCCESS

**Reply by donsez** [reply]
Hi,

You can try the package semtech_loramac into RIOT OS with the following boards : nucleo-wl55jc and lora-e5-dev

https://github.com/RIOT-OS/RIOT/tree/master/boards/nucleo-wl55jc
https://github.com/RIOT-OS/RIOT/tree/master/boards/lora-e5-dev
https://github.com/RIOT-OS/RIOT/tree/master/tests/pkg_semtech-loramac

Port has been done by @fjmolinas

I had (successful) used it for those balloon benchmarks (433 MHz and 868 MHz)
https://gricad-gitlab.univ-grenoble-alpes.fr/thingsat/public/-/tree/master/balloons/2021-04-15#media
with
https://github.com/CampusIoT/orbimote/tree/master/field_test_device

Kind regards

Didier

**Reply by gilbertoe** [reply]
Hi
Thanks for the info. I wil try them.
Is there any way to configure the lora-mac-node to support the STM32WLE5CCU6 ?
BR
Gilberto
